Assignment6.Rmd

```diff
@@ -16,25 +16,27 @@ assignment (numeric) - A student's average grade for the course assignments
 #Packages
 ```{r}
 library(rpart)
+library(dplyr)
+library(tidyr)
 ```
 
 #Data
 ```{r}
 #Upload the data sets MOOC1.csv and MOOC2.csv
 M1 <- read.csv("MOOC1.csv", header = TRUE)
 
-M2 <- 
+M2 <- read.csv("MOOC2.csv", header=T)
 
 ```
 
 #Decision tree
 ```{r}
 #Using the rpart package generate a classification tree predicting certified from the other variables in the M1 data frame. Which variables should you use?
 
-c.tree1 <- 
+c.tree1 <- rpart(certified ~ forum.posts + grade + assignment, method="class", data=M1, control = rpart.control(minsplit = 1,minbucket = 1,cp=0.001))
 
 #Check the results from the classifcation tree using the printcp() command
```

Comment:
Good work 👍 Though you may need to `scale()` the variables so that `forum.posts` does not swamp the other two since it has many more increments